settings.ini

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
```

Comment:
Does an other package use `settings.ini` file for this?

Maybe we should just expand on setup.cfg

https://github.com/pypa/setuptools/blob/master/setup.cfg

Reply:
I know Numpy does, so too my package Caer, but I suppose there isn't much difference using a `settings.ini` file over expanding over the `setup.cfg`. Needless, I've removed `settings.ini` and moved all configurations to `setup.cfg`.

Reply:
Can we reference numpy's `settings.ini` or `setup.cfg` file by any chance? We typically follow their lead on certain things like this.

Reply:
I must have got mixed up with another package. Working with far too many these days! [Fastai](https://github.com/fastai/fastai/blob/master/settings.ini) has a `settings.ini` file to separate out meta information from `setup.py`. While they use a `settings.ini` file, I suggest sticking with setup.cfg (since that's where the configuration variables are defined).